CHANGELOG.md

```diff
@@ -4,7 +4,22 @@
 breaking changes in the upcoming 3.x release. This release is scheduled for
 2024-12 or 2025-01.
 
-## v2.33.0 - TBD
+## v2.33.0 - 2024-12
+
+### [BigQuery Control](/google/cloud/bigquerycontrol/README.md)
+ - Has been promoted from an `experimental` library to GA.
+
+### [Storage](/google/cloud/storage/README.md)
+
+- docs(storage): Update build instructions for gcs+grpc ([#14833](https://github.com/googleapis/google-cloud-cpp/pull/14833))
+
+### [Common Libraries](/google/cloud/README.md)
+
+- fix(gkeconnect): service only supports REST endpoint ([#14897](https://github.com/googleapis/google-cloud-cpp/pull/14897))
```

Comment:
Should be in its own GKE Connect section, not Common

Reply:
Done.
<!-- Reviewable comment -OEQ_wfyDm_MNgP7TPde:b-896fix -->
<!-- Sent from Reviewable.io -->